src/TapeChart.js

```diff
@@ -0,0 +1,48 @@
+class TapeChart {
```

Comment:
Tell me about choosing the name for this class?  Is this a parent class?

Reply:
I named it TapeChart because it is essentially performing the calculations on the data for rooms and bookings for the hotel's big picture, so-to-speak.  There is a tool in hotel pms systems called the tape chart that does/displays this so it just makes sense to my hotel-brain.

I don't have this in mind as a parent class currently but I've had a much harder time getting a clear plan for class structure for this project.  I anticipate that I'll have to stay flexible and adjust my class structure as I see it all coming to life.